1.0/docs/devguide/styling.md

```diff
@@ -24,92 +40,148 @@ provided via `<style>` tags placed inside the element's local DOM `<template>`.
+DOM. So when you use your element:
 
-    Becomes:
+    <my-element>
+        <div class="special">This should be special!</div>
+    </my-element>
 
-        x-foo div
+Only `This should be special!` will have an orange background.
 
-    (Where `x-foo` is the name of the custom element.)
+#### Shim limitations
 
-*   To limit styles to elements inside the ::content tag, add a wrapper element around the
-    `<content>` element. This is especially important when using a child combinator (`>`) to
-    select top-level children.
+Shady DOM is much more limited than shadow DOM.
 
-        <dom-module id="my-element">
+Under shady DOM,  the `<content>` tag doesn't appear in the DOM tree, and styles
+are rewritten to remove the `::content` pseudo-element **as well as and any
+combinator immediately to the left of `::content`.** (E.g. `> :: content` where `>`
+is the combinator before it).
 
-          <template>
+Using shady DOM, the example above sees the style `::content > .special {`
+is rewritten as `my-element > .special`. This means that the rule will not apply
+just to the light DOM (as it should), but to the local DOM too (which it
+shouldn't).
 
-            <style>
-              .content-wrapper ::content > .special {
-                background: orange;
-              }
-            </style>
+**NOTE: The original documentation is very confusing here, as it says that you must have a selector before `::content` (for example `:host`), but in my experience that doesn't actually do anything since BOTH `:host ::content div` and `::content div` becomes `element div` (which is obviously not enough). I wonder if adding `element` was a recent-ish addition, and earlier versions of polymer had `::content div` turn into a dangerous, leaky `div`!**
 
-            <div class="content-wrapper"><content></content></div>
+This means that if you want to target only and only the light DOM, you will need
+to wrap `<content></content>` in a `div`, and target that specific `div` in your
+rules:
 
-          </template>
+    <dom-module id="my-element">
 
-        </dom-module>
+      <template>
+
+        <style>
+          .content-wrapper ::content > .special {
+            background: orange;
+          }
+        </style>
+
+        <div class=".special">I am not special!</div>
+        <div class="content-wrapper"><content></content></div>
+
+      </template>
 
-    In this case, the rule:
+      <script>
+          Polymer({
+              is: 'my-element'
+          });
+      </script>
+
+    </dom-module>
 
-        .content-wrapper ::content > .special
+In this case, the rule:
 
-    Becomes:
+    .content-wrapper ::content > .special
 
-        .content-wrapper > .special
+Becomes:
 
+    .content-wrapper.my-element  > .special
 
+**NOTE: I have a bit of a problem here as the chapter hasn't yet introduced**
+**custom properties, and yet it warns "Custom properties can't style distributed**
+**children. I am commenting this warning out for now**
```

Comment:
This is true, but it's also true that people don't tend to read docs in order. This is a large enough limitation (and a big enough source of confusion) that I wanted to put it up front here.